lisa/wlgen/rta.py

```diff
@@ -492,7 +492,9 @@ def get_rtapp_repr(self, task_name):
                              running_time, sleep_time)
 
             phase['loop'] = cloops
-            phase['run'] = running_time
+            # rtapp fails to handle floating values correctly
+            # https://github.com/scheduler-tools/rt-app/issues/82
+            phase['run'] = int(running_time)
```

Comment:
Might want to do that for `sleep_time` and `running_time` above directly, so that we always log int values as well.